app/javascript/styles/application.scss

```
@@ -18,6 +18,10 @@
   color: govuk-colour($colour: "white");
 }
 
+.govuk-panel--grey {
```

```suggestion
.gray-background {
```
In such case, we can use this style not only for panel